ang/crmMailing/EditRecipCtrl.js

```diff
@@ -73,9 +73,7 @@
     // refresh setting at a duration on 5sec
     var refreshSetting = _.debounce(function() {
       $scope.$apply(function() {
-        crmApi('Setting', 'getvalue', {"name": 'auto_recipient_rebuild', "return": "value"}).then(function(response) {
-          $scope.permitRecipientRebuild = (response.result === 0);
-        });
+        $scope.permitRecipientRebuild = ($scope.$parent.crmMailingConst.autoRecipientRebuild == 0 || $scope.$parent.crmMailingConst.autoRecipientRebuild === false);
```

Comment:
I'm not sure I understand the logic here. `0` and `false` are both falsey, so this should be sufficient if it absolutely must be converted to boolean (although I'm not sure that's completely necessary)
```suggestion
        $scope.permitRecipientRebuild = !!$scope.$parent.crmMailingConst.autoRecipientRebuild;
```

Reply:
@colemanw I was finding that I think the previous code was expecting an explicit 0 which is what you might get from the API but Civi::settings() was giving me false but let me test your option locally

Reply:
@colemanw I found that this worked if I only used 1 `!` not 2

Reply:
Oh right because the original code said `== 0`